sql/amo_prod/fenix_addons_by_client_v1/query.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TEMP function get_fields(m ANY TYPE) AS (
+  STRUCT(
+    m.submission_timestamp,
+    m.client_info.client_id,
+    m.sample_id,
+    m.metrics.string_list.addons_enabled_addons,
+    m.client_info.app_build,
+    m.normalized_country_code,
+    m.client_info.locale,
+    m.normalized_os
+  )
+);
+
+WITH unioned AS (
+  SELECT
+    get_fields(m1).*
+  FROM
+    org_mozilla_fenix.metrics AS m1
+  UNION ALL
+  SELECT
+    get_fields(m2).*
+  FROM
+    org_mozilla_fenix_nightly.metrics AS m2
+  UNION ALL
+  SELECT
+    get_fields(m3).*
+  FROM
+    org_mozilla_fennec_aurora.metrics AS m3
+  UNION ALL
+  SELECT
+    get_fields(m4).*
+  FROM
+    org_mozilla_firefox.metrics AS m4
+  UNION ALL
+  SELECT
+    get_fields(m5).*
+  FROM
+    org_mozilla_firefox_beta.metrics AS m5
+),
+per_client AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_id,
+    sample_id,
+    array_concat_agg(addons_enabled_addons) AS addons,
+    udf.mode_last(array_agg(app_build)) AS app_version,
```

Comment:
Just to double check, this will use the app build ID while for `desktop_addons_by_client_v1` it uses the version.

Reply:
Yes, it's my understanding that `app_build` is the Glean/Fenix equivalent of app_version. We put it in a field named `app_version` for schema compatibility with the desktop table.

Reply:
@jklukas looking at some production data, `app_build` seems to be the build ID (like `2015752523`), which is kinda mysterious for end users. [This patch](https://hg.mozilla.org/integration/autoland/rev/46949083a8c8) shows a `app_display_version` field and I am wondering if this could be something we could used instead?

I haven't tried to query any data yet, so that's just a question.

Reply:
That's definitely worth investigating. If `app_display_version` is a more useful quantity for you, I'm happy to use that instead.

Reply:
Let's revisit later (when Fenix will be GA).